setup.py

```diff
@@ -16,8 +15,10 @@
 # dist info so setuptools can't find it, even though its there, which results in
 # pkg_resources.DistributionNotFound, even though the package is available. So we
 # only added it if we aren't building with conda.
+# fabio from the hexrd channel has a similar problem.
```

Comment:
I have removed fabio from our channel, so I think this comment can me removed.

Reply:
I'll give it a try

Reply:
Yep, I just gave it a try, and this change is no longer needed. I'll revert this commit.

Reply:
@cjh1 That commit was removed, and now we are back to README only changes.